Kibana/Exports/kibana-7-10.ndjson

```diff
@@ -0,0 +1,25 @@
+{"exportedCount":24,"missingRefCount":0,"missingReferences":[]}
```

Comment:
Shall we add a newline. Look for other places as well

Reply:
Fixed with https://github.com/wso2-extensions/identity-elk-integration/pull/14